gitgraph.js

```diff
@@ -125,7 +125,20 @@ var gitGraph = function (canvas, rawGraphList, config) {
 			!(row[i - 2] && row[i] === "_" && row[i - 2] === "|")) {}
 		
 		return i;
-	}
+	};
+	
+	var findLineBreak = function (row) {
+		if (!row) {
```

Comment:
I think there should also need to have check ` || row.length === 0` just to be sure not to have infinite loop below in unexpected corner case

Reply:
postfix decrement in the while loop returns 0 what is false and stops it, and row.length can't be negative